app/helpers/application_helper/button/miq_ae_domain_priority_edit.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationHelper::Button::MiqAeDomainPriorityEdit < ApplicationHelper::Button::Basic
+  def disabled?
+    (User.current_tenant.visible_domains.length < 2)
+  end
```

Comment:
You'll need one more routine to set a button tooltip for the disabled situation:

```ruby
def calculate_properties
  super
  self[:title] = N_("You need two or more domain to edit domain priorities") if disabled?
end
```